VelocityPulse/XV
# [INTERACTIONS] [UIBUBLEINFO]

* System of subscriptions for each Interaction
* Add Script on Item prefab:
Generic Interaction class for item that don't have interaction but need to be interactable
Item that have specific Interaction / Animation will have their own class, Example: ChariotPlateforme
* Update UIBubbleInfo prefab to make button scrollable, in anticipation of new animation buttons.
* UIBubbleInfoButton are tagged, this allow us to delete specific button, using their tag
* Add manage function to UIBubbleInfoButton using their tag, (Delete, ContainsButton, ...)

# Info:
* Only ChariotPlateforme have draft of Interaction implementation for now
# To test if all work well: 
* Run XV, Add ChariotPlateforme to your scene, then add an HEAVY_ITEM like Caisse1m3
Expected: Two Button appear 

* Remove the HEAVY_ITEM
Expected: Interaction buttons disappear

#### Draft of Animation creation in : ChariotPlateforme.cs
